tests/test_cycluslib.py

```diff
@@ -3,20 +3,17 @@
 import subprocess
 from functools import wraps
 
-import nose
-from nose.tools import assert_equal, assert_less
-
 from cyclus import lib
 
 from tools import libcyclus_setup, dbtest
 
 
-setup = libcyclus_setup
+#setup = libcyclus_setup
```

Comment:
Can we remove this line?

Reply:
I've put it back in and am testing.  I can't recall why we dropped it?